### DISCUSSION TITLE
Merge react-nexus into recoil

### DISCUSSION THREAD

**Discussion by richardanaya** [facebookexperimental/Recoil#1816] [Ideas]
I'm pretty confused why this basic functionality isn't present in the library.  There's appears many people that find react-nexus's idea useful.  Looking inside it's code I see no unstable usages.

**Comment by luisanton-io** [comment]
I suppose @richardanaya was referring to [Recoil Nexus](https://www.npmjs.com/package/recoil-nexus) which lately appears to have gained some traction.
Surely if the Recoil team confirms this is ok I will provide a PR

EDIT: A pull request #1826 is now open.
